### ISSUE TITLE
Auctions not Starting but Etherscan shows Bid

### ISSUE THREAD

**Issue by Laramie43ip** [ethereum/ens-registrar-dapp#160]
I've been noticing some significant problems today.  I've been using MyEtherWallet (switched from Metamask a few weeks back) to bid. I initiated this transaction for the name "latisha" just over three days ago:

Here's the transaction hash: 0xa3c81e71252fde603512ea274e643d3ef920d9126512aa351764e0008dd541b1

Here it is on Etherscan: https://etherscan.io/tx/0xa3c81e71252fde603512ea274e643d3ef920d9126512aa351764e0008dd541b1

The bid amount is correct.  The transaction did not fail.  I copied the hash to reveal the bid.  All info appears to be correct.  I made a separate copy of the three word secret phrase.  It matches the info in the hash used to reveal.  The owner account is correct.  That auction should be in the reveal phase now since it's over 3 days old.  But, it isn't.

This has happened with multiple bids place 3+ days ago, all of which should be in the reveal phase today.  I have transaction hashes and hashes to use to reveal for each.  The transaction hashes all show up on Etherscan, funds all went to the ENS contract, but no auctions were initiated.  It can't be misspelled words because everything I do is cut and paste -- both to initiate auctions and to reveal.

What becomes of these funds?  I can see that they went to the ENS contract, but I will not be able to reveal because the auction never commenced.

**Comment by Laramie43ip** [comment]
No help on this, but I thought I would report back what happened.  As mentioned in the post above, there were four separate auctions where Etherscan showed the bid but the auction did not launch.  Oddly, they were all roughly within an hour of each other.

Two of the auctions were subsequently launched by others.  When they got to the reveal phase, my bids were over three days old (so, the auction reveal phase did not correspond to when I submitted my bid).  In the reveal phase, I was able to reveal.  So, even though the auctions weren't started with my bid -- indeed, for these two names in particular they didn't start at all until well after my bid -- still, my stale bid was accepted in the reveal phase.

Since then, I "re-launched" the other two auctions for de minimis amounts.  I'll report back on whether I can reveal both bids since two bids were now submitted for each name.  Quite odd, but I thought I would update in case it can help anyone else with the same issues.

**Comment by Arachnid** [comment]
That's correct - bids are valid for 19 days, and it doesn't matter when they were submitted, as long as it was before the reveal period starts. You don't need to pay anything other than gas fees to start an auction, though.

**Comment by Laramie43ip** [comment]
Turns out there are still problems.  To summarize: I initiated the auction on "latisha."  It as not recognized even though I had a confirmed transaction hash.  Someone else then launched the auction for that name.  When it got to the reveal phase, I was able to reveal, and I have a reveal hash.

But, I am not able to finalize.  The transaction shows the auction as "final" and the contract owner as blank.  When I try to finalize with the account used for bidding, I get the following error: "This account is not the owner of latisha.eth. Please unlock the Owner Account in order to resolve."

This same error also is given for the other name that did not launch, but for which I was able to reveal after someone else initiated the auction.  Thanks in advance for any help.

**Comment by Arachnid** [comment]
If you're trying to reveal right now, there is a large transaction backlog on the network, which will take a couple of hours to clear.

**Comment by Laramie43ip** [comment]
Got it.  BAT-related, I suspect.  Hope you guys got in.  You deserve it.  Thanks for all the great effort!
